test/extended/operators/clusteroperators.go

```diff
@@ -67,6 +73,48 @@ var _ = g.Describe("[Feature:Platform] ClusterOperators", func() {
 		})
 
 	})
+
+	g.Describe("fields validation", func() {
+		var (
+			oc                               = exutil.NewCLI("cluster-basic-auth", exutil.KubeConfigPath())
+			openshiftApiServerOperatorClient = oc.AdminOperatorClient().OperatorV1().OpenShiftAPIServers()
```

Comment:
can we move this into openshift/openshift-apiserver?

Reply:
Ok , I'll close this  and open a new one, thanks.